go/vt/mysqlctl/replication.go

```diff
@@ -228,6 +230,38 @@ func (mysqld *Mysqld) GetServerUUID(ctx context.Context) (string, error) {
 	return conn.Conn.GetServerUUID()
 }
 
+// GetGlobalStatusVars returns the server's global status variables asked for.
+// An empty/nil variable name parameter slice means you want all of them.
+func (mysqld *Mysqld) GetGlobalStatusVars(ctx context.Context, statuses []string) (map[string]string, error) {
+	query := "SELECT variable_name, variable_value FROM performance_schema.global_status"
```

Comment:
I still think it's worth making this a const in the package. With that, you could also use `fmt.Sprintf` to add only the `WHERE` clause to it in ParseAndBind. Then we don't have two related queries to keep in sync and we're not doing unnecessary allocations.

Reply:
Done